### DISCUSSION TITLE
Add README to NuGet packages

### DISCUSSION THREAD

**Discussion by loic-sharma** [LuckyPennySoftware/AutoMapper#3704] [Ideas]
Hello!

I am from the NuGet team at Microsoft. We recently introduced support for embedding READMEs into NuGet packages (blog, docs), and we plan to place these READMEs front and center on nuget.org. Would you be willing to accept a contribution to add embedded READMEs to your packages?

**Comment by lbargaoanu** [comment]
Sure. Thanks!

**Comment by loic-sharma** [comment]
I prepared the changes [here](https://github.com/AutoMapper/AutoMapper/compare/master...loic-sharma:patch-1?expand=1) but this is blocked by this bug: https://github.com/NuGet/Home/issues/10791
